theme/src/js/vendor/zendesk-default-script.js

```diff
@@ -2,147 +2,135 @@
+		commentContainerTextarea.focus();
+	});
 
-		requestMarkAsSolvedButton.on(
-			'click',
-			function () {
-				$('.request-container .comment-container input[type=checkbox]').attr('checked', true);
+	var requestCommentSubmitButton = commentContainer.find(
+		'input[type=submit]'
+	);
 
-				requestCommentSubmitButton.prop('disabled', true);
+	var requestMarkAsSolvedButton = requestContainer.find(
+		'.mark-as-solved:not([data-disabled])'
+	);
 
-				$(this).attr('data-disabled', true).closest('form').submit();
-			}
+	requestMarkAsSolvedButton.on('click', function() {
+		$('.request-container .comment-container input[type=checkbox]').attr(
+			'checked',
+			true
 		);
 
-		var requestCommentTextarea = requestContainer.find('.comment-container textarea');
-
-		requestCommentTextarea.on(
-			'keyup',
-			function() {
-				var text = requestMarkAsSolvedButton.data('solve-translation');
+		requestCommentSubmitButton.prop('disabled', true);
 
-				var hasText = requestCommentTextarea.val() !== '';
+		$(this)
+			.attr('data-disabled', true)
+			.closest('form')
+			.submit();
+	});
 
-				if (hasText) {
-					text = requestMarkAsSolvedButton.data('solve-and-submit-translation');
-				}
+	var requestCommentTextarea = requestContainer.find(
+		'.comment-container #hc-wysiwyg'
+	);
 
-				requestCommentSubmitButton.prop('disabled', !hasText);
+	requestCommentTextarea.on('keyup', function() {
+		var text = requestMarkAsSolvedButton.data('solve-translation');
 
-				requestMarkAsSolvedButton.text(text);
-			}
-		);
+		var hasText = requestCommentTextarea.val() !== '';
 
-		if (requestCommentTextarea.val() === '') {
-			requestCommentSubmitButton.prop('disabled', true);
+		if (hasText) {
+			text = requestMarkAsSolvedButton.data(
+				'solve-and-submit-translation'
+			);
 		}
 
-		$('.collapsible-nav, .collapsible-sidebar').on(
-			'click',
-			function(event) {
-				event.stopPropagation();
+		requestCommentSubmitButton.prop('disabled', !hasText);
 
-				setExpanded(this);
-			}
-		);
+		requestMarkAsSolvedButton.text(text);
+	});
 
-		$('.header .icon-menu').on(
-			'click',
-			function(event) {
-				event.stopPropagation();
+	if (requestCommentTextarea.val() === '') {
+		requestCommentSubmitButton.prop('disabled', true);
+	}
 
-				var userNav = document.getElementById('user-nav');
+	$('.collapsible-nav, .collapsible-sidebar').on('click', function(event) {
+		event.stopPropagation();
 
-				setExpanded(userNav);
-			}
-		);
+		setExpanded(this);
+	});
 
-		$('.share a').click(
-			function(event) {
-				event.preventDefault();
+	$('.header .icon-menu').on('click', function(event) {
+		event.stopPropagation();
 
-				window.open(this.href, '', 'height=500, width=500');
-			}
-		);
+		var userNav = document.getElementById('user-nav');
 
-		$('#quick-search').on(
-			'keypress',
-			function(event) {
-				if (event.which === 13) {
-					search();
-				}
-			}
-		);
+		setExpanded(userNav);
```

Comment:
Should we check if `userNav` returns a DOM node before using in JS?

Reply:
Sounds good

Reply:
Added at fd61930!